src/common-utils/main.sh

```diff
@@ -491,7 +491,7 @@ if [ "${INSTALL_ZSH}" = "true" ]; then
         echo 'source $HOME/.profile' >> "${user_home}/.zprofile" # TODO: Reconsider adding '.profile' to '.zprofile'
```

Comment:
Let's bump the Feature version and add a new test scenario (instead of updating the existing one) which validates the issue mentioned by the user in https://github.com/devcontainers/features/issues/1035

Reply:
version bumped to : 2.4.6
Also created a scenario `configure_zshrc_without_overwrite` to validate the issue mentioned by the user, kindly review it once.